exec/java-exec/src/test/java/org/apache/drill/test/QueryTestUtil.java

```diff
@@ -47,9 +46,6 @@
  * Utilities useful for tests that issue SQL queries.
  */
 public class QueryTestUtil {
-
-  public static final String TEST_QUERY_PRINTING_SILENT = "drill.test.query.printing.silent";
```

Comment:
If we are going to remove this property, I think it should be also removed from `pom.xml` file as well. Please also check for references in other places.

Reply:
Grepped for it and removed references in pom.xml and Testing.md